bzkanban.js

```diff
@@ -222,6 +225,31 @@ function createBacklogTarget() {
     return backlog;
 }
 
+function createBacklogButton() {
+    var backlogShowButton = document.createElement("button");
+    backlogShowButton.id = "btnShow" + backlogTitle;
+    backlogShowButton.innerText = "Show Backlog";
+    backlogShowButton.addEventListener("click", function() {
+        var button = document.getElementById("btnShow" + backlogTitle);
```

Comment:
Also, noticed that the state of the button will need to get reset when switching products.